src/cli/cmds/fleet.toit

```diff
@@ -455,6 +470,14 @@ init parsed/cli.Parsed config/Config cache/Cache ui/Ui:
         --broker-config=broker-config
         --ui=ui
 
+login parsed/cli.Parsed config/Config cache/Cache ui/Ui:
+  with-pod-fleet parsed config cache ui: | fleet/Fleet |
+    broker := fleet.broker
+    broker-name := broker.server-config.name
+    ui.info "Logging in to broker '$broker-name'."
```

Comment:
Sometimes broker/server names are printed in single quotes like here, but in other cases it is printed with no quotes. I'd prefer it to be consistent.

Reply:
I will do a follow-up PR.